etc/omero.properties

```diff
@@ -150,6 +150,15 @@ omero.glacier2.IceSSL.VerifyPeer=0
 # Glacier2Template SSL allowed protocols
 omero.glacier2.IceSSL.Protocols=tls1
 
+# Override the default set of OMERO servers.
```

Comment:
Are these `OMERO servers` or `services`?

Reply:
If it was me I'd call them services, but the ice teminology is `server`:
```
$ omero admin ice server list
Blitz-0
DropBox
FileServer
Indexer-0
MonitorServer
OMERO.Glacier2
OMERO.IceStorm
PixelData-0
Processor-0
Tables-0
TestDropBox
```
There's a separate `omero admin ice service ...` command but I can't work out what it does. What do you reckon is best?

Reply:
Pretty sure services only work in IceBox which we don't use: https://doc.zeroc.com/ice/3.7/icebox

Reply:
Yes, even in the same paragraph below it's "distribute OMERO services across multiple nodes" which is confusing.

Reply:
No one else said anything so I've changed it to `services`